### PR TITLE
Development

### PR DESCRIPTION
**Pull request by asc2683**

### CODE REVIEW

**Comment by asc2683** [comment]
This is ready to be merged.

File: front-end/src/components/forms/TeamMemberForm.js [hunk 1]
```diff
@@ -92,17 +114,69 @@ class TeamMemberForm extends React.Component {
               </Checkbox>
             </FormGroup>
 
-            <SelectTeam name="team" teams={this.props.teams} selected={this.state.team} onSelect={(e) => {
-              this.onInputChange(e);
-            }} />
+            <ControlLabel>Experience</ControlLabel>
+            <FormGroup controlId="experience">
+              <Radio
```
Comment (keithb418) [inline front-end/src/components/forms/TeamMemberForm.js:119]: This group of radio buttons could be generated in a loop.  That would cut down on the lines of code and code reuse

File: front-end/src/components/forms/SelectTeam.js [hunk 2]
```diff
@@ -5,19 +5,19 @@ import PropTypes from "prop-types";
...
       <FormControl
         name="team"
         defaultValue={selected}
       	componentClass="select"
       	placeholder="Select Team"
       	onChange={onSelect} >
-    
-        <option>Select Team...</option>
+
+        <option>Select Team</option>
```
Comment (keithb418) [inline front-end/src/components/forms/SelectTeam.js:16]: The value on this could be set to "" so that the form validation is simpler.

**Comment by asc2683** [comment]
I am closing this pull request since the pull request should've been made to the development branch.

**Pull request closed without merging**